index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://fonts.googleapis.com/css?family=Poppins"
+      rel="stylesheet"
+    />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+  <body>
+    <header>
+      <nav>
+        <a href="#">My Logo</a>
+
+        <img id="menu-icon" src="images/Enabled.png" alt="" />
+      </nav>
+    </header>
+
+    <main>
+      <h1>Article Title</h1>
+      <p>
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed euismod
+        euismod nisi, ac bibendum odio bibendum vel. Donec euismod nisl ut magna
+        aliquam, vel lacinia sapien bibendum. Sed eget velit vel velit lacinia
+        dapibus.
+      </p>
```

Comment:
nice work so far but your text needs to be a custom text not a dummy text and also please aling your text as the design